src/main/java/ladder/controller/LadderController.java

```diff
@@ -42,18 +45,52 @@ public Players readPlayers() {
         return new Players(playerNames);
     }
 
+    private Prizes readPrizes(final int playerCount) {
+        final String readPrizes = inputView.readPrizes();
+        final List<String> prizes = Converter.stringToList(readPrizes);
+
+        return new Prizes(prizes, playerCount);
+    }
+
     public Height readLadderHeight() {
         final int height = inputView.readLadderHeight();
 
         return new Height(height);
     }
 
-    private void printLadder(final Players players, final Ladder ladder) {
+    private void showInformationForLadderGame(final Players players, final Ladder ladder, final Prizes prizes) {
         outputView.printLadderResultMessage();
+
+        showPlayerNames(players);
+        showLadder(ladder);
+        showPrizes(prizes);
+    }
+
+    private void showPlayerNames(final Players players) {
         outputView.printPlayerNames(PlayersDto.from(players));
+    }
+
+    private void showLadder(final Ladder ladder) {
         outputView.printLadder(LadderDto.from(ladder));
     }
 
+    private void showPrizes(final Prizes prizes) {
+        outputView.printPrizes(prizes.getPrizes());
+    }
+
+    private void showResult(final PlayResult playResult) {
+        String name = inputView.readNameToSeeResult();
+
+        while (!Command.EXPRESSION_OF_ALL_PLAYER.isMatch(name)) {
+            final String result = playResult.findPlayerResultByName(name);
```

Comment:
```suggestion
        while (!Command.isMatchAll(name)) {
            final String result = playResult.findPlayerResultByName(name);
```

Enum의 상수값과 직접 비교하기 보다는 Enum 클래스에게 물어보는 방식이 좋아보여요